mathics/core/evaluation.py

```diff
@@ -161,7 +163,7 @@ def __init__(self, definitions=None,
         self.predetermined_out = None
 
         self.quiet_all = False
-        self.quiet_messages = set()
+        self.quiet_messages = QUIET_MESSAGES
```

Comment:
What happens with `QUIET_MESSAGES` in a multi-user session? Does one user's `Off` affect everyone?